docs/loadjdbc.adoc

```diff
@@ -490,3 +497,14 @@ CALL apoc.load.jdbc('jdbc:derby:derbyDB','SELECT * FROM PERSON WHERE NAME = ?',[
 ----
 2018-10-31T01:32:25.012+09:00[Asia/Tokyo]
 ----
+
+
+.with credentials
+[source,cypher]
+----
+CALL apoc.load.jdbcUpdate('jdbc:derby:derbyDB','UPDATE PERSON SET NAME = ? WHERE NAME = ?',['John','John'],{credentials:{user:'apoc',password:'Ap0c!#Db'}})
```

Comment:
Documentare bene che si può fare in entrambi i modi sia come hai scritto tu, sia così:
`jdbc:derby:derbyDB;user=apoc;password=Ap0c!#Db;create=true`